src/main/java/core/basesyntax/SupplyOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class SupplyOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Storage storage) {

```

Comment:
Ensure that the `handle` method implementation here follows the SOLID principles. In this context, it looks fine, but always be cautious about single responsibility and open-closed principles when adding more logic.